### PR TITLE
TFM net8.0 and removal of net7.0 was: TFM net8.0

### PR DESCRIPTION
**Pull request by moljac**
`net7.0-android` hit EOL on [May 14, 2024](https://dotnet.microsoft.com/en-us/platform/support/policy/maui) and thus we will no longer support it in our AndroidX binding packages.   Dropping `net7.0-android` will allow us to support packages that require API-34 like newer versions of Google Material going forward.

Update packages to use `net8.0-android` instead of `net7.0-android`.

Note that the "removed" `published-namespaces` are namespaces that only contained an empty `Resources` class that is no longer generated.

Samples changes:

- migrated samples to SDK style projects
- NuGet Cetntral Package Management
- collecting of Nugets - minor changes
- Cake build script - minor changes

Old description:

> ### Does this change any of the generated binding API's?

> Probably. Not sure yet. There were some Metadata changes.

> ### Describe your contribution

> In May support for Xamarin.Classic (`monoandroid12`) and `net6.0` will be dropped and this is recon PR to see what is ahead of the team.

### CODE REVIEW

File: samples/com.google.android.play/play-services-assetpack/AssetPackSample/MainActivity.cs [hunk 1]
```diff
@@ -79,7 +79,7 @@ private void FabOnClick(object sender, EventArgs eventArgs)
 
 		public override void OnRequestPermissionsResult(int requestCode, string[] permissions, [GeneratedEnum] Android.Content.PM.Permission[] grantResults)
 		{
-			Xamarin.Essentials.Platform.OnRequestPermissionsResult(requestCode, permissions, grantResults);
+			Microsoft.Maui.ApplicationModel.Platform.OnRequestPermissionsResult(requestCode, permissions, grantResults);
```
Comment (dellis1972) [inline samples/com.google.android.play/play-services-assetpack/AssetPackSample/MainActivity.cs:82]: Wow, we have to pull in Maui essentials to do this?

Comment (moljac) [reply samples/com.google.android.play/play-services-assetpack/AssetPackSample/MainActivity.cs:82]: Well, I "only" migrated Xamarin.Essentials code to MAUI Essentials.

Your samples do build, so I see no big issue. Even better we do test how everything works with MAUI side.

File: samples/com.google.android.play/play-services-assetpack/AssetPackSample/AssetPackSample.csproj [hunk 2]
```diff
@@ -129,13 +55,5 @@
       <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
     </ProjectReference>
   </ItemGroup>
-  <Import Project="$(MSBuildExtensionsPath)\Xamarin\Android\Xamarin.Android.CSharp.targets" />
-  <Import Project="..\Targets\DynamicFeature.targets" />
```
Comment (dellis1972) [inline samples/com.google.android.play/play-services-assetpack/AssetPackSample/AssetPackSample.csproj:133]: This is needed for the dynamic features to work.

Comment (dellis1972) [reply samples/com.google.android.play/play-services-assetpack/AssetPackSample/AssetPackSample.csproj:133]: Also missing the `AssetPacks` and Targets folders in the directory above.

Comment (dellis1972) [reply samples/com.google.android.play/play-services-assetpack/AssetPackSample/AssetPackSample.csproj:133]: So all of this https://github.com/xamarin/GooglePlayServicesComponents/tree/main/samples/com.google.android.play/play-services-assetpack

**Review by jpobst — APPROVED** [review]

**Comment by dellis1972** [comment]
@moljac 

We are still missing the required targets/projects etc for the Asset Pack sample by the looks of things.

**Comment by moljac** [comment]
> @moljac
> 
> We are still missing the required targets/projects etc for the Asset Pack sample by the looks of things.

@dellis1972 

Just to leave comment here, because it is confusing: that was UI not showing change[s] - targets/projects are not missing.

**Comment by jpobst** [comment]
Closing as GPS packages have been moved to https://github.com/xamarin/AndroidX repository and this repository is no longer used.

**Pull request closed without merging**